apps/frontend/src/components/global/upload_tabs/aws/FileList.vue

```diff
@@ -78,6 +79,14 @@ export default class FileList extends Vue {
     // Fetch it from s3, and promise to submit it to be loaded afterwards
     await fetch_s3_file(this.auth.creds, file.Key!, this.formBucketName).then(
       (content) => {
+        try {
+          JSON.parse(content);
```

Comment:
I don't think we should add this restriction such that only JSON files can be processed from S3 buckets.  It'd be nice to support any filetype (ex. any of the XML based results files) that could be stored in a bucket and then converted on load.

Reply:
Currently the code only processes JSON files, once we lift that we can change it here. The problem is that if you have any other format in the S#, there isn't any notification to the user of any problems, and the file doesn't load.